packages/patterns/src/types.js

```diff
@@ -493,10 +493,17 @@
 
 /**
  * @typedef {any} MethodGuardMaker
- * a parameter list like foo(a, b, c = d, …e) => f should be guarded by
- * something like
- * foo: M.call(AShape, BShape).optional(CShape).rest(EShape).returns(FShape)
- * optional is for optional (=) params. rest is for … (varargs) params
+ * A method name and parameter/return signature like:
+ * ```js
```

Comment:
I'm not sure what agoric-sdk file the corresponds to, so I'm not sure how to verify that this is just syncing up.

Reply:
https://github.com/Agoric/agoric-sdk/blob/841a7012831166f6c29f00bdffc907f2b35a7b0c/packages/store/src/types.js#L754-L767

Resolving.